From 7cdbff926324c7855f4ca5af816dfe24c562acf3 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:32 -0400
Subject: [misc] mei: txe: include irqreturn.h for irqreturn_t etc

Message-id: <1399041470-21330-60-git-send-email-prarit@redhat.com>
Patchwork-id: 79345
O-Subject: [RHEL7.1 PATCH BZ 1088953 59/77] mei: txe: include irqreturn.h for irqreturn_t etc
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 4a22176aa89b8121a5160064d77244e26fe38790
Author: Stephen Rothwell <sfr@canb.auug.org.au>
Date:   Fri Feb 21 16:38:28 2014 +1100

    mei: txe: include irqreturn.h for irqreturn_t etc

    Signed-off-by: Stephen Rothwell <sfr@canb.auug.org.au>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/hw-txe.c b/drivers/misc/mei/hw-txe.c
index 19579e5..49a5ed3 100644
--- a/drivers/misc/mei/hw-txe.c
+++ b/drivers/misc/mei/hw-txe.c
@@ -18,6 +18,7 @@
 #include <linux/jiffies.h>
 #include <linux/delay.h>
 #include <linux/kthread.h>
+#include <linux/irqreturn.h>
 
 #include <linux/mei.h>
 
diff --git a/drivers/misc/mei/hw-txe.h b/drivers/misc/mei/hw-txe.h
index 857d88c..fdb665f 100644
--- a/drivers/misc/mei/hw-txe.h
+++ b/drivers/misc/mei/hw-txe.h
@@ -17,6 +17,8 @@
 #ifndef _MEI_HW_TXE_H_
 #define _MEI_HW_TXE_H_
 
+#include <linux/irqreturn.h>
+
 #include "hw.h"
 #include "hw-txe-regs.h"
 
-- 
1.7.1